ome/ome-ngff-validator
# Multiple viewers

Fixes #19.

To test: deployed at https://deploy-preview-23--ome-ngff-validator.netlify.app/?source=https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0062A/6001240.zarr 

cc @joshmoore This adds a JSON file where you can configure the various viewers for `Open with...`;

I've tried to add logos from the various viewers, but they don't all have logos - neuroglancer had no logo (just used a micro screenshot) and I used an AICS logo for their viewer, which is not ideal.

![Screenshot 2023-02-17 at 10 51 21](https://user-images.githubusercontent.com/900055/219627741-b991e0c7-a370-47f1-8b7a-858d37439502.png)

To test:
 - Links to web-based viewers should open viewers in new tab and display the image
 - Mousing over other icons (napari, MoBIE) gives instructions and shows a 'copy' icon to copy the image URL
